packages/core/elasticsearch/core-elasticsearch-client-server-internal/src/client_config.test.ts

```diff
@@ -125,12 +125,44 @@ describe('parseClientOptions', () => {
           false,
           kibanaVersion
         );
-        expect(options.agent).toHaveProperty('maxSockets', 1024);
+        expect(options.agent).toHaveProperty('maxTotalSockets', 1024);
       });
 
       it('defaults to `Infinity` if not specified by the config', () => {
         const options = parseClientOptions(createConfig({}), false, kibanaVersion);
-        expect(options.agent).toHaveProperty('maxSockets', Infinity);
+        expect(options.agent).toHaveProperty('maxTotalSockets', Infinity);
+      });
+    });
+
+    describe('`maxIdleSockets` option', () => {
```

Comment:
Same here. Looking at other tests' describe blocks, it seems that the description refers to the input parameter (in this case `maxIdleSockets`. So far, so good.

But then, in the `it(...)` block I'd update the description to make it clear that this parameter translates to `maxFreeSockets` output parameter.

Also valid for `idleSocketTimeout => timeout`.